src/NUnitFramework/framework/Internal/PlatformHelper.cs

```diff
@@ -245,6 +246,13 @@ public bool IsPlatformSupported( string platform )
                 case "WIN8.1":
                     isSupported = os.IsWindows81;
                     break;
+                case "WINDOWS10":
+                case "WIN10":
+                    isSupported = os.IsWindows10;
+                    break;
+                case "WINDOWSSERVER":
```

Comment:
I think consistency with our own terminology may be more important than conforming to MS terminology here. WindowsServer in the context of PlatformAttribute sounds like it should include all prior versions of server as well. I'd go with Windows10Server or WindowsServer10.